### ISSUE TITLE
The encoding of the API response?

### ISSUE THREAD

**Issue by junhoyeo** [zhuowei/ClubhouseAPI#6]
The API keep returning weird strings as non-alphabet values, such as `ğŸ˜´`(😴), `ğŸ‡«ğŸ‡·`(🇫🇷), `Letâ€™s`(Let's).
Do anyone have/figured out/know the solution about this issue? 🙏

**Comment by zhuowei** [comment]
It should be UTF-8 (special characters shows up fine on my macOS machine, which uses UTF-8 by default).

This looks like you're decoding the response with the [Windows-1252](https://stackoverflow.com/a/2477480) encoding, but I see that you're [specifying UTF-8](https://github.com/junhoyeo/graphclubhouse/blob/master/notebook.ipynb) in your code, so I'm not sure what you're missing. Can you link to your code where you're seeing this issue?

**Comment by junhoyeo** [comment]
@zhuowei First, sorry for the late response and feedback.

1. [My Python code with `requests` and all that](https://github.com/junhoyeo/graphclubhouse/blob/master/notebook.ipynb)(and yes, that's the link you found above) was running in macOS too, and that's UTF-8 by default. It kept returning strange mojibake.
2. I rewrite this in Node.js and `axios` and it started working. LOL.

Well I have nothing to do father, since I used the Python script first only to test the APIs, and you provided information with the encodings, so I'll just close this issue.

In case if someone encounters the same issue that I've experienced, I'll leave some references...
- My question on Stack Overflow: [How to decode this broken emoji string(mojibake) with python or something?](https://stackoverflow.com/questions/66172188/how-to-decode-this-broken-emoji-stringmojibake-with-python-or-something)
- Running Node.js code
```js
const axios = require('axios');

(async () => {
  const { data } = await axios.post(
    'https://www.clubhouseapi.com/api/get_profile',
    {
      user_id: USER_ID,
    },
    {
      headers: {
        Authorization: `Token ${AUTH_TOKEN}`,
        'CH-UserID': USER_ID,
        'User-Agent': 'clubhouse/269 (iPhone; iOS 14.1; Scale/3.00)',
        'CH-Languages': 'en-US',
        'CH-Locale': 'en_US',
        'CH-AppVersion': '0.2.15',
        'CH-AppBuild': '269',
        'CH-DeviceId': UUID.toUpperCase(),
      },
    }
  );
  console.log(JSON.stringify(data));
})();
```
